### PR TITLE
Adding CognitiveSearch.Skills project

### PR DESCRIPTION
**Pull request by dereklegenzoff**
Feel free to let me know if any changes are needed

### CODE REVIEW

**Comment by LuisCabrer** [comment]
@dereklegenzoff , please remove binary files like .vs/CognitiveSearch.Template/v15/Server/sqlite3/storage.ide-shm or the .suo file (those are files that include your visual studio preferences, etc)

File: CognitiveSearch.Skills/.gitignore [hunk 1]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (LuisCabrer) [inline CognitiveSearch.Skills/.gitignore]: Please remove this file from your checkin, as well as other binary files.

File: CognitiveSearch.Skills/host.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (LuisCabrer) [inline CognitiveSearch.Skills/host.json:1]: Why is this file needed?

File: CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json [hunk 3]
```diff
@@ -1,9 +1,8 @@
 {
   "runtimeOptions": {
```
Comment (LuisCabrer) [inline CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json]: Maybe we should delete this file while we are at it.

File: CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json [hunk 4]
```diff
@@ -1,9 +1,8 @@
 {
   "runtimeOptions": {
     "additionalProbingPaths": [
-      "C:\\Users\\luisca\\.dotnet\\store\\|arch|\\|tfm|",
-      "C:\\Users\\luisca\\.nuget\\packages",
-      "C:\\Program Files (x86)\\Microsoft SDKs\\NuGetPackagesFallback",
+      "C:\\Users\\delegenz\\.dotnet\\store\\|arch|\\|tfm|",
+      "C:\\Users\\delegenz\\.nuget\\packages",
```
Comment (LuisCabrer) [inline CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json]: We may want to delete this file too...

**Review by LuisCabrer — COMMENTED** [review]
Please see my comments. Thanks!

File: CognitiveSearch.Skills/host.json (hunk 2 above)
Comment (dereklegenzoff) [reply CognitiveSearch.Skills/host.json:1]: This is a file that gets created when you create an azure function project in VS. Based on the [docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json), it holds various settings about the azure function. In many cases, it'll just specify that version 2 is used. Looks to me like it get's updated after publishing so it may make sense to keep the file around

File: CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json (hunk 4 above)
Comment (dereklegenzoff) [reply CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json]: I've removed this file in my most recent commit.

File: CognitiveSearch.Skills/.gitignore (hunk 1 above)
Comment (dereklegenzoff) [reply CognitiveSearch.Skills/.gitignore]: removed

**Comment by dereklegenzoff** [comment]
> @dereklegenzoff , please remove binary files like .vs/CognitiveSearch.Template/v15/Server/sqlite3/storage.ide-shm or the .suo file (those are files that include your visual studio preferences, etc)

Thanks, Luis! I removed these files in the latest commit
